assets/aurora.tokenlist.json

```diff
@@ -77,6 +77,17 @@
         "ethereum"
       ]
     },
+    {
+      "chainId": 1313161554,
+      "address": "0x5ac53f985ea80c6af769b9272f35f122201d0f56",
+      "symbol": "HAK",
+      "name": "Hakuna Matata",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/HAK.svg",
+      "tags": [
+        "ethereum"
+      ]
+    },
```

Comment:
Don't update that manually, this will be updated by Aurora team once PR is accepted and the metadata for the token is updated by us.